Bindings/Python/simple-arm.py

```diff
@@ -0,0 +1,132 @@
+
+shoulder = osim.PinJoint("shoulder",
+	arm.getGround(), 	    		# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+elbow = osim.PinJoint("elbow",
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	radius, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+# ---------------------------------------------------------------------------
+# Add a muscle that flexes the elbow (actuator for robotics people).
+# ---------------------------------------------------------------------------
+
+biceps = osim.Millard2012AccelerationMuscle("biceps", 	# 	Muscle name
+   	200.0, 	 	#	Max isometric force
+    	0.6, 		# 	Optimal fibre length
+    	0.55, 		# 	Tendon slack length
+    	0.0)		# 	Pennation angle
+biceps.addNewPathPoint("origin",						
+   	humerus, 						
+   	osim.Vec3(0, 0.8, 0))			
+
+biceps.addNewPathPoint("insertion",
+   	radius,
+   	osim.Vec3(0, 0.7, 0))
+
+# # ---------------------------------------------------------------------------
+# # Add a controller that specifies the excitation of the muscle.
+# # ---------------------------------------------------------------------------
+
+brain = osim.PrescribedController()
+brain.addActuator(biceps)
+brain.prescribeControlForActuator(1,					
+	osim.StepFunction(0.5,3.0,0.3,1.0))
+	
+# ---------------------------------------------------------------------------
+# Build model with components created above.
+# ---------------------------------------------------------------------------
+
+arm.addBody(humerus)
+arm.addBody(radius)
+arm.addJoint(shoulder)			# Now required in OpenSim4.0 
+arm.addJoint(elbow)				
+arm.addForce(biceps)
+arm.addController(brain)
+
+# ---------------------------------------------------------------------------
+# Add a console reporter to print the muscle fibre force and elbow angle.
+# ---------------------------------------------------------------------------
+
+# We want to write our simulation to a file in the end
+reporter = osim.TableReporter()				
+reporter.set_report_time_interval(1.0)
+reporter.updInput("inputs").connect(biceps.getOutput("fiber_force"))
+reporter.updInput("inputs").connect(elbow.getCoordinateSet().get(0)\
+	.getOutput("value"), "elbow_angle")
+arm.addComponent(reporter)
+
+# ---------------------------------------------------------------------------
+# Configure the model.
+# ---------------------------------------------------------------------------
+
+state = arm.initSystem()
+# Fix the shoulder at its default angle and begin with the elbow flexed.
+arm.updCoordinateSet().get(0).setLocked(state, True) 
+arm.updCoordinateSet().get(1).setValue(state, 0.5 * osim.SimTK_PI)
+arm.equilibrateMuscles(state)
+
+# ---------------------------------------------------------------------------
+# We don't care about graphical simulation so we won't do that.
+# Python API currently does not support forward simulation.
```

Comment:
Can remove these two lines. I realized that you *can* do forward simulation using the ForwardTool. We should just explain at the top that this is a model-building example.